api/base/filters.py

```diff
@@ -292,13 +292,28 @@ def parse_query_params(self, query_params):
                             field_name: self._parse_date_param(field, source_field_name, op, value),
                         })
                     elif not isinstance(value, int) and source_field_name in ['_id', 'guid._id', 'journal_id', 'moderation_state']:
```

Comment:
I think we need to separate `['_id', 'guid._id', 'journal_id', 'moderation_state']` into two separate case:
*  `'_id', 'guid._id'` should go through the version flow as you've implemented
* `'journal_id', 'moderation_state'` stay the same as before